packages/platforms/inventec/x86-64/x86-64-inventec-d7032q28b/onlp/builds/src/module/auto/x86_64_inventec_d7032q28b.yml

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+#
+# x86_64_inventec_d7032q28b Autogeneration Definitions.
+#
+###############################################################################
+
+cdefs: &cdefs
+- x86_64_inventec_d7032q28b_CONFIG_INCLUDE_LOGGING:
```

Comment:
All of these config defines are by convention uppercase. 
Please s/x86_64_inventec_d7032q28b_CONFIG/X86_64_INVENTEC_D7032Q28B_CONFIG/g and regenerate.